pipeline/extract.py

```diff
@@ -131,13 +131,26 @@ def get_case_title(case_soup: BeautifulSoup) -> str:
 def download_pdfs(court_case: dict) -> None:
     """Downloads a pdf from the link given in the court_case dict."""
 
-    if not path.exists(f"/{ENV['STORAGE_FOLDER']}"):
+    if not path.exists(f"{ENV['STORAGE_FOLDER']}"):
         makedirs(f"{ENV['STORAGE_FOLDER']}")
 
-    court_case["filepath"] = f"/{ENV['STORAGE_FOLDER']}/{court_case['title']}.pdf"
-    response = requests.get(court_case["pdf"])
-    with open(f"{court_case['filepath']}", "wb") as f:
-        f.write(response.content)
+    court_case["filepath"] = f"{ENV['STORAGE_FOLDER']}/{court_case['title']}.pdf"
+
+    try:
+        response = requests.get(court_case["pdf"], timeout=10)
+        with open(f"{court_case['filepath']}", "wb") as f:
+            f.write(response.content)
+        return
+    except (requests.exceptions.MissingSchema, requests.exceptions.ReadTimeout):
+        pass
```

Comment:
nice try and except, what dos the except check for however?

Reply:
It's a check in case the url given is invalid or the request times out